### DISCUSSION TITLE
How to configure circuit breaker in celery

### DISCUSSION THREAD

**Discussion by SerenaCapsule** [celery/jumpstarter#38] [General]
Celery 5 introduces the concept of Ciruit Breaker into the framework here.

Anyone knows how to configure the circuit breaker ? An example will be appreciated!

**Comment by auvipy** [comment]
no it didn't yet. it will in future.

**Comment by thedrow** [comment]
What was Celery 5 became nextgen.
This repository is the foundation for nextgen.
I'd appreciate any help.

**Reply by auvipy** [reply]
Omer can you link architectural part to discussion section as well? I have plans to start going over the design docs again and prepare some async related Dep, from by the end of this month to first week of next month.

**Reply by thedrow** [reply]
> 
> 
> Omer can you link architectural part to discussion section as well? I have plans to start going over the design docs again and prepare some async related Dep, from by the end of this month to first week of next month.

I'll find it in the relevant CEP.
We should schedule a video chat. It's long overdue.

**Reply by auvipy** [reply]
lets try next week, if i am available.
